index.js

```diff
@@ -31,7 +32,7 @@ Sentry.init({
   // We recommend adjusting this value in production.
   environment: getEnvironmentName(),
   tracesSampleRate: 1.0,
-  release: '1.11.6',
+  release: `${code[0]}.${code[1]}.${code[2]}`,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The way you're constructing the release string from the `code` array assumes that the array always has at least three elements. If it doesn't, this will cause an error. It would be safer to check the length of the array before accessing its elements.
```diff
-  release: `${code[0]}.${code[1]}.${code[2]}`,
+  release: code.length >= 3 ? `${code[0]}.${code[1]}.${code[2]}` : 'unknown',
```
This change ensures that if the `code` array does not have at least three elements, the release is set to 'unknown'.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
done on next commit